net-analyzer/ettercap/ettercap-0.8.3.1-r2.ebuild

```diff
@@ -56,6 +56,7 @@ BDEPEND="
 
 PATCHES=(
 	"${FILESDIR}"/${P}-curl-8.patch
+	"${FILESDIR}"/${P}-musl.patch #897820
```

Comment:
Implicit function declaration fixes actually require a revbump, I'd just `git mv` here.

Reply:
Good point, I forget these aren't always errors... I made the change.